File: R/graph.R
install.packages("ggplot2")
library("ggplot2")

dat <- read.csv("/Users/danyabaron/Desktop/CS480x/02-DataVis-5Ways/cars-sample.csv")

print(dat)

# Create plot object
scatterGraph.plot <- ggplot(data = dat, 
                       mapping = aes(x = Weight, y = MPG)) +

geom_point(aes(size=Weight), color=Manufacturer, alpha=0.5) +
scale_color_manual(values = c("red", "orange", "forestgreen", "darkblue", "violet"))


# Draw plot
print(scatterGraph.plot)

File: python/graph.py
import pandas as pd
import matplotlib
import matplotlib.pyplot as plt
import numpy as np

print('hello')
data  = pd.read_csv("cars-sample.csv")

# df.plot()
# df.plot(kind='scatter',x='Weight',y='MPG')

colors = {'ford': '#a6e39f', 'bmw': 'red', 'honda': 'green', 'mercedes': '#92e0f0', 'toyota': '#f092ce'}

plt.xlabel("Weight")
plt.ylabel("MPG")
plt.scatter(data["Weight"], data["MPG"], s=(data["Weight"] / 25), alpha=0.5, c=data["Manufacturer"].map(colors))

plt.show()






















